### ISSUE TITLE
The generation result of the bg pseudo-label should not be mutually exclusive, but softmax is used instead.

### ISSUE THREAD

**Issue by regedit818** [caoluyang0830/CVBM#4]
In your code:
```
plab_a_bg = get_cut_mask(unoutput_a_bg, nms=1)
plab_b_bg = get_cut_mask(unoutput_b_bg, nms=1)

def get_ACDC_masks(output, nms=0,onehot=False):
    probs = F.softmax(output, dim=1)
    _, probs = torch.max(probs, dim=1)
    if nms == 1:
        if onehot:
            probs = get_ACDC_2DLargestCC_onehot(probs)
        else:
            probs = get_ACDC_2DLargestCC(probs)
    return probs
```
It is correct for binary classification tasks. But, this is disastrous for the ACDC of background pseudo-labels. Should the use of sigmoid be considered?
